blog/_posts/2017-09-13-Blog-1team-Serv0101.md

```diff
@@ -0,0 +1,38 @@
+We just can’t believe that RGSoC is coming to an end, time flies. 
+
+### Learning and Challenges
+
+We were absolutely unfamiliar with Rust, and it has quite a bit of a learning curve. We struggled with Rust.  You need a lot of patience when you are learning something new it really takes a lot of time. We faced a lot of problems while understanding the codebase, specification, building environment etc. But yeah, Kudos to our coaches Rahul, Ravi and our Mentor Josh they were always up for help that was the only thing helped us in keep going. We both believe, we have learned a lot of things during last two months and still learning.
+
+### Project Journey
+* Initially, setting up the project on our local machines was easy but we faced the problem when it took more than 1 hr 30 minutes to build. Then our mentor came to rescue us by introducing to a tool called Janitor.
+* Small Intro to Janitor: The janitor is a web app which uses cloud9. Instead of installing, downloading servo in your machine with lots of instruction you can simply go to janitor where complete development environment is provided for Mozilla firefox, servo etc. You can simply go there and chose servo and started editing in cloud9. Thank you, Jan Keromnes for creating this cool stuff.
+* Everyday we got introduced to some new concepts of Rust programming language. We are glad that we choose this project because we are really loving Rust.
+* One more challenge was understanding the specification. The language used in the specification was hard at first to understand but with help of our mentor and coaches, it was easy.
+* During the review of PR, we got to know that there are few bugs in the specification for which Josh opened the issue.
+* Matching the standard of code in the servo was tough for both of us at the beginning then with the suggestion of our coaches we started writing our own code and then after the suggestion of mentor and coaches we refactored the code which is matching the codebase of servo.
+* So now we came up with this: We wrote more than 50 logs(more to come).
+* Number of comments on Github: 122(more to come). We discussed most of the things on IRC but still, we are getting lots of comments and suggestion from different developers. It’s really nice to see other community members are taking interest in our project and helping us. That’s why people say `open source` is really filled with cool and awesome people.
+* Number of Files created + changed: 15.
+* We both were familiar with Git before the starting of the project but in last two months believe us we messed up something while rebasing and learned some new things about it. Big thanks to coach Rahul and Ravi(They both were our pillars of strength and support)- you people are awesome.
```

Comment:
Change 'with Git before the starting of the project'  -> 'with a Git before starting the project'. Add comma before and after 'believe us' .